src/genie/libs/parser/ios/show_vlan_internal_usage.py

```diff
@@ -0,0 +1,65 @@
+'''
+Author: Mehdi Cherifi
+Twitter: https://twitter.com/LocketKeepsake
+Github: https://github.com/cherifimehdi
+
+'''
+import re
+
+# Metaparser
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any
+
+
+# ====================================================
+#  Schema for 'show vlan internal usage' command
+# ====================================================
+class ShowVlanInternalUsageSchema(MetaParser):
+    """Schema for:
+        show vlan internal usage"""
+
+    schema = {
+        'Internal Vlan': {
+            Any(): {
+                        'Usage': str,
+            },
+        }
+    }
+
+# =================================================
+# Parser for 'show vlan internal usage' command
+# =================================================
+class ShowVlanInternalUsage(ShowVlanInternalUsageSchema):
```

Comment:
This parser and its schema should be under `show_vlan.py`, rather than creating a new file for it. Additionally, for `ios` devices which are typically identical to `iosxe` devices, we'll usually put the parser under `iosxe` and import it into `ios`. You can see an example of what I mean in `src/genie/libs/parser/ios/show_vlan.py`

Reply:
please should I remove the file show_vlan_internal_usage.py and pull directly my change under show_vlan.py?